tripal/src/TripalImporter/TripalImporterBase.php

```diff
@@ -614,4 +614,30 @@ public function getArguments() {
     return $this->arguments;
   }
 
+  /**
+   * Validates whether XML is valid or not.
+   *
+   * @param string $xml
+   *   The XML to be checked.
+   *
+   * @return bool
+   *   Return TRUE if valid, FALSE if not valid.
+   */
+  protected function xmlIsValid(string $xml): bool {
+    $valid = TRUE;
+
+    libxml_use_internal_errors(TRUE);
+    $doc = simplexml_load_string($xml);
+    if ($doc === FALSE) {
+      $valid = FALSE;
+    }
+    else {
```

Comment:
```suggestion
    // Enable user handling of errors so that exceptions are not.
    libxml_use_internal_errors(TRUE);
    // Attempt to load the XML.
    $doc = simplexml_load_string($xml);
    // If SimpleXML fails to parse the XML string then it will return FALSE.
    if ($doc === FALSE) {
      $valid = FALSE;
    }
    // If not, we will check for any errors logged during parsing.
    else {
```